src/main/aciotest/main.c

```diff
@@ -53,11 +55,15 @@ static bool aciotest_assign_handler(
     }
 
     if (product_type == AC_IO_NODE_TYPE_BI2A) {
+        printf("Please assign bi2a mode, 0 for SDVX and 1 for IIDX, then press ENTER:");
```

Comment:
I guess that works, but compared to the other ways it detects different devices, it feels a bit off.
Do we ever have multiple acio devices on the same bus with any BI2A device? To my knowledge, the BI2A device is always on its own bus exclusively (please correct me here if I am wrong).
Assuming that's the case, I suggest to extend the tool rather to allow selecting different "kinds" of devices.

Examples how to call `aciotest`:
* `aciotest.exe COM1 57600` for a "generic" acio device that does the entire "go by product code" thing
* `aciotest.exe COM1 57600 bi2a-iidx` for the iidx BI2A mode
* `aciotest.exe COM1 57600 bi2a-sdvx` for the sdvx BI2A mode

Mind tweaking the tool a little to have it work like that?

Reply:
Sorted!